sms/send-sms.php

```diff
@@ -1,24 +1,22 @@
 <?php
+
+use Zend\Diactoros\Response\Serializer;
+use Zend\Diactoros\Request\Serializer as asdf;
+
 require_once __DIR__ . '/../config.php';
 require_once __DIR__ . '/../vendor/autoload.php';
 
 $basic  = new \Nexmo\Client\Credentials\Basic(NEXMO_API_KEY, NEXMO_API_SECRET);
 $client = new \Nexmo\Client($basic);
 
-try {
-    $message = $client->message()->send([
-        'to' => TO_NUMBER,
-        'from' => 'Acme Inc',
-        'text' => 'A text message sent using the Nexmo SMS API'
-    ]);
-    $response = $message->getResponseData();
+$response = $client->sms()->send(
+    new \Nexmo\SMS\Message\SMS(TO_NUMBER, BRAND_NAME, 'A text message sent using the Nexmo SMS API')
+);
 
-    if($response['messages'][0]['status'] == 0) {
-        echo "The message was sent successfully\n";
-    } else {
-        echo "The message failed with status: " . $response['messages'][0]['status'] . "\n";
-    }
-} catch (Exception $e) {
-    echo "The message was not sent. Error: " . $e->getMessage() . "\n";
-}
+$message = $response->current();
 
+if($message->getStatus() == 0) {
```

Comment:
This is a minor thing, but in some examples, we have `if(` and in others, we have a space between them. Would it be an idea to have one or the other for consistency?

Reply:
PSR-12 states there should be a space, so it should always be a space. I agree on the consistency. For v3.0.0 snippets I want to run everything through `phpstan` and `phpcs-fixer` to get all the stylish things fixed.